src/test/resources/test-schema.sql

```diff
@@ -7,4 +7,17 @@ CREATE TABLE IF NOT EXISTS items (
                        picture_url VARCHAR,
                        weight INTEGER,
                        stock_amount INTEGER
-);
\ No newline at end of file
+);
+
+CREATE TABLE categories (
+                            id BIGINT PRIMARY KEY,
+                            name VARCHAR(255) NOT NULL,
+                            parent_category_id INT,
+                            FOREIGN KEY (parent_category_id) REFERENCES category(id)
+);
+
+ALTER TABLE items
```

Comment:
In the test folder, you don't need to alter previously created tables.
Just edit the CREATE statement, it will be easier to understand.